src/web/UnitTests/Domain/ReadModel/ApprenticeshipFavouriteTests.cs

```diff
@@ -39,5 +44,19 @@ public void IsFramework_ReturnFalse_WhenStandard(string id)
 
             Assert.False(sut.IsFramework);
         }
+
+        //[Fact]
+        //public void GenerateAddress_SelectsPrimaryAddress()
+        //{
+        //    List<ProviderAddress> Address = new List<ProviderAddress> { new ProviderAddress { ContactType = "LEGAL" },
+        //                                                                new ProviderAddress { ContactType = "PRIMARY" } };
+        //    var generator = new ();
+        //}
+
+        [Fact]
+        public void GenerateAddress_WhenAllInformationIsAvailable()
+        {
+
+        }
```

Comment:
Is this yet to be implemented?

Reply:
This was something I was looking at implementing but now it maps each element of the address and joins them in the view model instead.